apps/hefty/lib/hefty.ex

```diff
@@ -179,4 +179,25 @@ defmodule Hefty do
     )
     |> Hefty.Repo.one()
   end
+
+  def fetch_transactions(offset, limit, symbol \\ "") do
+    Logger.debug("Fetching transactions for a symbol", symbol: symbol)
```

Comment:
could you please stop using metadata for data that you would like to include in the message - it was a mistake and should be fixed wherever happened before - it should now look like:
```
Logger.debug("Fetching transactions for a symbol(#{symbol})")
```

Reply:
Sorry.